mkdocs.yml

```diff
@@ -176,33 +177,32 @@ extra_css:
 extra_javascript:
       - https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js
       - javascripts/tablesort.js
+      - javascripts/cell-toggles.js
 
 # Custom home page
 home:
       # Custom Homepage Elements
-      tagline: Our data workflow
+      tagline: Our World in Data - Technical documentation
       # cover_image: https://github.com/tylerdotrar/mkdocs-coverpage/blob/main/docs/attachments/wizard-posting.png?raw=true #attachments/wizard-posting.png # This path starts at your root mkdocs directory.
-      description: |
-            Find all the details about OWID's data management tool.
+      description: |-
+            This documentation explains our data workflow and tooling — both to help OWID team members work effectively and to give external readers insight into how we manage and publish our data.
+            You can also find selected public analyses related to our work.
       # Gradient Color
       # gradient_hsla: hsla(173, 100%, 24%, 1) # Default Green: hsla(160deg,47%,55%,1)
 
       buttons:
-            - name: Get started
+            - name: Data workflow
               url: getting-started
-              # icon: people
               theme: md-button md-button--primary
-              # blurb: This joker navigates to the shadow wizard armory located in "/Folder1/index.md"
-            - name: Learn more
-              url: architecture
-              # icon: gear
-              theme: md-button
-              # blurb: This goober navigates to some important thingies located in "/Folder2/index.md"
-            - name: Guides
+            # - name: Learn more
+            #   url: architecture
+            #   theme: md-button
+            - name: Internal guides
               url: guides
               theme: md-button
-              # blurb: This psycho navigates to an interesting table of stuff located in "/Folder3/index.md"
-              # icon: image
+            - name: Public analyses
```

Comment:
I don't like "Public analyses", I think it's a confusing term. I'd rather use something like "Technical publications", to clearly highlight what these are: publications on technical matters. I don't think we need to use the word "public", as everything on our docs page is indeed public.